@xen-orchestra/web-core/lib/components/menu/MenuTrigger.vue

```diff
@@ -28,7 +28,6 @@ defineProps<{
   padding-right: 1.5rem;
   padding-left: 1.5rem;
   white-space: nowrap;
-  border-radius: 0.8rem;
   gap: 1rem;
   background-color: var(--color-neutral-background-primary);
```

Comment:
```suggestion
.menu-trigger {
  display: flex;
  align-items: center;
  padding-inline: 1.6rem;
  gap: 0.8rem;
```
According to figma you can even simplify like that